dockerfiles/packaging/debian/package/etc/james/cassandra.properties

```diff
@@ -0,0 +1,6 @@
+# Configuration file for cassandra mailbox
+cassandra.nodes=cassandra
```

Comment:
I just want to ask you a question here such as the guy from system team in VN had to install James on Centos yesterday and I see you missed documentation.
Could we have a mention somewhere to notify that james service should be started in case of cassandra and elasticsearch services are ready at host "cassandra" and "elasticsearch"?